### ISSUE TITLE
Need help

### ISSUE THREAD

**Issue by MohamedElidrissi** [nirenr/AndroLua_pro#2]
Hey this is not really an issue and I know this is a very old project but I've been trying to use the text editor you used and I have problem with setting up the lexer, so I used jflex to generate a java lexer class but its alot different than the one you have (LuaLexer) can you tell me how did you generate it?

**Comment by AndroidScave** [comment]
Estimation is generated by JFlex.

**Comment by MohamedElidrissi** [comment]
@AndroidScave Yeah I know but when I generate a java lexer with jflex I get a class which is different than the one in this project therefore I can't use it

**Comment by debatebug** [comment]
@MohamedElidrissi  Maybe @nirenr  forgot the GitHub code. More than two years did not push the project to GitHub. But a new version has been released in a chat software.

**Comment by debatebug** [comment]
@MohamedElidrissi I found items that I am interested in in your repositories, **KIDE-Kotlin-IDE-for-Android** ,Are you trying to develop a kotlin IDE that runs on android?

**Comment by MohamedElidrissi** [comment]
@debatebug yes indeed I'm still working on it. It requires root to run a script and requires a custom jre 1.8 and kotlinc packages (approx 150mb) and it compiles even a hello world program pretty slow on my Galaxy J7 Prime. Even though I decided to develop it completely on my phone using AIDE app I can get my current progress uploaded now if you're interested to contribute

**Comment by MohamedElidrissi** [comment]
@debatebug "But a new version has been released in a chat software"
Any link where I can find that?

**Comment by AndroidScave** [comment]
@MohamedElidrissi 
http://www.imqq.com/mobile/mobile-page.html

**Comment by MohamedElidrissi** [comment]
@AndroidScave So after I log in how can I access that new version?

**Comment by debatebug** [comment]
@MohamedElidrissi 
group account:236938279 , You can find group in that software through group account number; latest source code in Group file
@nirenr Account number is :946049229 , You can find him in that software through his account number;
Or I can help you get The latest source code.

**Comment by AndroidScave** [comment]
@MohamedElidrissi 
group account:236938279 , You can find group in that software through group account number; latest source code in Group file
@nirenr Account number is :946049229 , You can find him in that software through his account number;
Or I can help you get The latest source code.

**Comment by debatebug** [comment]
About IDE, I also have a project recently that needs some help. Have you ever tried using gradle on Android phone?

**Comment by AndroidScave** [comment]
I hope you can add my QQ, and we will have more exchanges.
My qq number is :1589206218

**Comment by MohamedElidrissi** [comment]
@AndroidScave I'm not really using QQ do you have anything else?

**Comment by debatebug** [comment]
I try to use WhatsApp to add your friend, but it Toast that the function is not supported. now i try WhatsApp PC Version.... wait

**Comment by MohamedElidrissi** [comment]
@debatebug you need to add me in your contacts list to show up

**Comment by debatebug** [comment]
I have uninstalled my contact software before. ...wait a few minutes。。。。=_=
